consensus/istanbul/backend/backend.go

```diff
@@ -383,15 +379,7 @@ func (sb *backend) GetProposer(number uint64) common.Address {
 
 // ParentValidators implements istanbul.Backend.GetParentValidators
 func (sb *backend) ParentValidators(proposal istanbul.Proposal) istanbul.ValidatorSet {
-	if block, ok := proposal.(*types.Block); ok {
-		return sb.getValidators(block.Number().Uint64()-1, block.ParentHash())
-	}
-
-	// TODO-Klaytn-Governance The following return case should not be called. Refactor it to error handling.
-	return validator.NewValidatorSet(nil, nil,
-		istanbul.ProposerPolicy(sb.governance.Params().Policy()),
-		sb.governance.Params().CommitteeSize(),
-		sb.chain)
+	return sb.getValidators(proposal.Number().Uint64()-1, proposal.ParentHash())
```

Comment:
Are there any possibilities for proposal not to be compatible with Block type? Go language handles the type properly, doesn't it?

Reply:
I'm not sure as to what would happen if proposal is not `*types.Block`. Initially I thought parser would have done the verification but not 100% sure. fixed in c830fbec